sdk/schemaregistry/azure-schemaregistry-avroserializer/README.md

```diff
@@ -99,7 +115,7 @@ fully_qualified_namespace = os.environ['SCHEMAREGISTRY_FULLY_QUALIFIED_NAMESPACE
 group_name = "<your-group-name>"
 
 schema_registry_client = SchemaRegistryClient(fully_qualified_namespace, token_credential)
-serializer = AvroSerializer(client=schema_registry_client, group_name=group_name)
+serializer = AvroSerializer(client=schema_registry_client, group_name=group_name, auto_register_schemas=True)
```

Comment:
Is there a  reason why we want it to be True?

because it makes feel like we want users to always turn on this feature, and if it's the case, shouldn't we be considering making this option default to true within out SDK?

Reply:
that's fair, maybe I can add a snippet where we register it first with the client? and don't pass in the auto_register_schemas=True?

Reply:
I'm not sure why the decision was made to not auto_register_schemas by default, but I think maybe it was because registering schemas is makes a call to the service. If we were to set this to True by default, that feels like silent behavior. The user may not want to make so many calls to the service, especially if they only have the one schema to register but lots of data to serialize for it.

Reply:
I get the point, it helps the onboarding experience if users haven't registered the schema into SR already.
let's keep `auto_register_schemas=True` for now.